### ISSUE TITLE
Even changes persist error (please no close until it's OK) --> INTEGER() can only be applied to a 'integer', not a 'NULL'

### ISSUE THREAD

**Issue by OmarVivas** [catboost/catboost#1626]
Problem: INTEGER() can only be applied to a 'integer', not a 'NULL'
catboost version: 0.25
Operating System: Windows
CPU: Yes
# GPU:

**When you run the test script**
_nonsense <- c('A', 'B', 'C')
data_frame <- data.frame(value = runif(10), category = nonsense[(1:10) %% 3 + 1])
label = (1:10) %% 2
pool <- catboost.load_pool(data_frame, label = label, cat_features = c(2))_

I get this error:
**Error in catboost.from_matrix(as.matrix(preprocessed), label, cat_features,  : 
  INTEGER() can only be applied to a 'integer', not a 'NULL'**

PS: Even install again, and fixing catboostr.cpp as david-cortes published.

Please, help us... I'm a Catboost fans but the last two version (0.24.4 and 0.25 don't work well)

**Comment by kizill** [comment]
We are preparinig hotfix release, will be available in a matter of hours.

**Comment by OmarVivas** [comment]
> We are preparinig hotfix release, will be available in a matter of hours.

Excellent, I'm looking forward it... CatBoost give me a very good model and performance in Kaggle competition's.

**Comment by OmarVivas** [comment]
> We are preparinig hotfix release, will be available in a matter of hours.

Hi @kizill after two days the error persist and dosen't work well yet... Please, help us... I will gonna try tomorrow and so on.

CatBoost is an excelent package but in R the last two version need to fix it.

Thanks you

**Comment by kizill** [comment]
Got unexpected problems with our build system, still fixing.

**Comment by OmarVivas** [comment]
> Got unexpected problems with our build system, still fixing.

Thanks @kizill, **I hope The CatBoost's Team can fix it soon in order to install the new version in R**.

By the way, I got a very good performance using CatBoost in R in this Kaggle competition -->  [https://www.kaggle.com/c/tabular-playground-series-apr-2021]

**Comment by kizill** [comment]
We've published fixed version in v0.25.1 release.
And please, don't forget to use `factor` function onto categorical variable to properly pass it to CatBoost :)
So your example should be read as:
```R
nonsense <- c('A', 'B', 'C')
data_frame <- data.frame(value = runif(10), category = factor(nonsense[(1:10) %% 3 + 1]))
label = (1:10) %% 2
pool <- catboost.load_pool(data_frame, label = label, cat_features = c(2))
```

**Comment by OmarVivas** [comment]
> We've published fixed version in v0.25.1 release.
> And please, don't forget to use `factor` function onto categorical variable to properly pass it to CatBoost :)
> So your example should be read as:
> 
> ```r
> nonsense <- c('A', 'B', 'C')
> data_frame <- data.frame(value = runif(10), category = factor(nonsense[(1:10) %% 3 + 1]))
> label = (1:10) %% 2
> pool <- catboost.load_pool(data_frame, label = label, cat_features = c(2))
> ```

Excellent!!!... Thanks you, I install and it is ok
